src/components/UserBar.tsx

```diff
@@ -11,7 +11,7 @@ export type UserBarProps = {
   avatar?: string;
   follow?: boolean;
   icon?: string;
-  onClickUser?: () => void;
+  onClickUser?: F<HTMLImageElement | SVGSVGElement | HTMLParagraphElement>;
```

Comment:
I feel like this can be more generalized to a simple HTML element
```suggestion
  onClickUser?: MouseEventHandler;
```